### PR TITLE
factoring out userProfileLibraryCard, giving it an isolate scope

### PR DESCRIPTION
**Pull request by 2is10**
@ahsu1230

### CODE REVIEW

**Comment by 2is10** [comment]
seems to work

File: kifi-backend/modules/shoebox/angular/src/userProfile/userProfileLibrariesList.tpl.html [hunk 1]
```diff
@@ -3,58 +3,8 @@
...
-      </div>
-      <div class="kf-upl-updated" ng-switch-when="updated">
-        Updated&nbsp;<time ng-attr-datetime="{{lib.lastKept}}" am-time-ago="lib.lastKept"></time>
-      </div>
-      <div class="kf-upl-unlisted" ng-switch-when="unlisted">
-        Hidden on public profile
-      </div>
-    </div>
+  <div kf-user-profile-library-card="library" ng-repeat="library in libraries"
+    current-page-name="{{::currentPageName}}" current-page-origin="{{::currentPageOrigin}}">
```
Comment (ahsu1230) [inline kifi-backend/modules/shoebox/angular/src/userProfile/userProfileLibrariesList.tpl.html:7]: oooo how do the `::` work?

Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/userProfile/userProfileLibrariesList.tpl.html:7]: It’s a magical prefix that means “only evaluate this binding once”.
https://docs.angularjs.org/guide/expression#one-time-binding

Comment (ahsu1230) [inline kifi-backend/modules/shoebox/angular/src/userProfile/userProfileLibrariesList.tpl.html:7]: I See. neat!

**Pull request merged**